pkg/api/internal/testing.go

```diff
@@ -210,7 +210,7 @@ func GetInfo(handler http.Handler) *httptest.ResponseRecorder {
 	return rec
 }
 
-func PostScript(t *testing.T, handler http.Handler, s core.Script, query url.Values) *httptest.ResponseRecorder {
+func PostScript(t *testing.T, handler http.Handler, s core.ScriptData, query url.Values) *httptest.ResponseRecorder {
```

Comment:
that seems strange to me because it looks like you're not giving the same object at all in the new test, can you explain why ?

Reply:
This is just a renaming. The reason is that we already have `core.TransactionData` that is containing almost the same information, so `core.ScriptData` seems appropriate.